plaso/output/shared_elastic.py

```diff
@@ -65,20 +65,27 @@ def __init__(self, output_mediator):
     self._password = None
     self._port = None
     self._username = None
+    self._ca_certs = None
 
   def _Connect(self):
     """Connects to an Elasticsearch server."""
-    elastic_hosts = [{'host': self._host, 'port': self._port}]
+    if self._port is not None:
+      elastic_hosts = [{'host': self._host, 'port': self._port}]
+    else:
+      elastic_hosts = [self._host]
 
     elastic_http_auth = None
     if self._username is not None:
       elastic_http_auth = (self._username, self._password)
 
     self._client = elasticsearch.Elasticsearch(
-        elastic_hosts, http_auth=elastic_http_auth)
+        elastic_hosts,
+        http_auth=elastic_http_auth,
+        ca_certs=self._ca_certs
```

Comment:
We'd also need to pass verify_certs=True, as I read the API: https://elasticsearch-py.readthedocs.io/en/master/connection.html?highlight=Connection

Reply:
verify_certs is True by default: https://github.com/elastic/elasticsearch-py/blob/master/elasticsearch/connection/http_requests.py#L30